smithy-diff/src/main/java/software/amazon/smithy/diff/evaluators/AddedEntityBinding.java

```diff
@@ -37,6 +38,8 @@
 public final class AddedEntityBinding extends AbstractDiffEvaluator {
     private static final String ADDED_RESOURCE = "AddedResourceBinding";
     private static final String ADDED_OPERATION = "AddedOperationBinding";
+    private static final String TO_RESOURCE = ".ToResourceShapeBound.";
+    private static final String TO_SERVICE = ".ToServiceShapeBound.";
```

Comment:
Open to suggestions if there are better wordings for this that don't also make the full id misleading